webapp/yarn.lock

```diff
@@ -0,0 +1,2159 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I usually donàt use yarn, I see that we have yarn lock file and package-lock.json file, I think we could upload only one of the two?